packages/console/src/containers/ConsoleContent/Sidebar/hook.tsx

```diff
@@ -30,10 +32,15 @@ type SidebarItem = {
 type SidebarSection = {
   title: TFuncKey<'translation', 'admin_console.tab_sections'>;
   items: SidebarItem[];
+  isHidden?: boolean;
```

Comment:
Maybe we don't need this prop, since it is only related to the `isClould`; we can use `isCloud` directly.
Also, we may need a switch for this WIP feature.